projects/1.20-fabric/assets/chatpatches/chatpatches/lang/zh_cn.json

```
@@ -0,0 +1,136 @@
+{
+	"modmenu.descriptionTranslation.chatpatches": "一个多功能的客户端聊天模组，让聊天更加定制化，查看配置和GitHub页面了解更多信息！\n我们的Discord是https://discord.gg/3mqbvneymz！",
```

原链接里的半角感叹号是链接的一部分吗
以下仅为建议，可不改
```suggestion
	"modmenu.descriptionTranslation.chatpatches": "一个多功能的客户端聊天模组，让聊天更加定制化，查看配置和GitHub页面了解更多信息！\n我们的Discord是https://discord.gg/3mqbvneymz ！",
```

---

不是